src/DocumentFormat.OpenXml.CodeGeneration.Linq/Program.cs

```diff
@@ -0,0 +1,536 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using DocumentFormat.OpenXml.Framework;
+using DocumentFormat.OpenXml.Framework.Metadata;
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace DocumentFormat.OpenXml.CodeGeneration.Linq
+{
+    public static class Program
+    {
+        private const string EmptyNamespace = "NoNamespace";
+
+        public static void Main()
```

Comment:
Is it a demo code?

Reply:
No, this is not meant to be demo code. This is a code generator that can be invoked on the command line or during the build process. It generates (most of) the classes in the `DocumentFormat.OpenXml.Linq` project.

Reply:
@ThomasBarnekow But I hard to find the directory: `const string directoryName = "..\\..\\..\\..\\src\\DocumentFormat.OpenXml.Linq\\GeneratedCode";`

Reply:
Do you mean that it is hard to find?

This might be a temporary way to specify the output path, which is the `GeneratedCode` folder within the `DocumentFormat.OpenXml.Linq` project. We need to first decide how the code generation is invoked and then determine how the path must be specified. But that would require some feedback from @twsouthwick.

Reply:
Yeah, we'll be playing around with this and hopefully be able to use a source generator soon. This is one of the big problems that source generators aim to solve, and I know who to bug if we have trouble getting them to work :)